xdsl/dialects/func.py

```diff
@@ -274,6 +272,10 @@ class CallOp(IRDLOperation):
     callee = prop_def(FlatSymbolRefAttrConstr)
     res = var_result_def()
 
+    assembly_format = (
+        "$callee `(` $arguments `)` attr-dict `:` functional-type($arguments, $res)"
+    )
+
     # TODO how do we verify that the types are correct?
```

Comment:
```suggestion
```

Reply:
My understanding was that this comment is essential #3497, no?
I was planning to tackle that next, but not sure when.

Reply:
ooh then yes